modules/LemDCBM400600/main/lem_dcbm_400600_controller.cpp

```diff
@@ -51,6 +51,33 @@ std::string LemDCBM400600Controller::get_current_transaction() {
     }
 }
 
+void LemDCBM400600Controller::update_lem_status() {
+    // should call this after a communication error to figure out what has been happening
+    auto status_response = this->http_client->get("/v1/status");
+
+    if (status_response.status_code != 200) {
```

Comment:
constant for 200

Reply:
btw, this code gets even more readable if you have a helper function like `bool status_ok(coonst StatusResponseType& status_response) { return status_response.status_code == STATUS_OK; }`